### PR TITLE
ci: Created .gitlab-ci.yml file (fix: #110)

### PR DESCRIPTION
**Pull request by psaavedra**
... implemented a simple smoke test unit to test the build process

### CODE REVIEW

**Comment by psaavedra** [comment]
Still WIP

**Comment by psaavedra** [comment]
> Still WIP

not anymore

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  - cd ~/ci-cog
+  - git clone $CI_REPOSITORY_URL
+  - cd cog
+  - git checkout $CI_COMMIT_REF_NAME
+
+cog-meta-perf-browser-warrior-wandboard-mesa-wpe-weston:
+  tags:
+    - docker
+  script:
+    - wget -O ~/toolchain.sh https://wk-contrib.igalia.com/yocto/meta-perf-browser/browsers/nightly/sdk/wandboard-mesa/browsers-glibc-x86_64-core-image-weston-wpe-armv7at2hf-neon-wandboard-mesa-toolchain-1.0.sh
```
Comment (aperezdc) [inline .gitlab-ci.yml]: As far as I can see those commands should run just fine on other GNU/Linux systems and nothing special is required. Why not putting those to run on Travis-CI, for example, or some other CI system which can run on GitHub PRs?

Comment (aperezdc) [reply .gitlab-ci.yml]: It just took me a few minutes to write down a basic `.travis.yml` using this toolchain, see PR #119 (and note how the CI ran on that PR already).

**Comment by psaavedra** [comment]
Just for curiosity sake. Is Travis CI free of charge?

**Comment by aperezdc** [comment]
> Just for curiosity sake. Is Travis CI free of charge?

Travis-CI is free (as in “free beer”) for Open Source projects hosted in publicly visible repositories. There is an “Enterprise” service with additional features which is paid.

[CircleCI](https://circleci.com) also has a free tier which supports using custom Docker images for the build environment, and supports more complex build pipelines. Travis is very easy to configure so that's why I tried it first, so I think we can use it for now because the features are enough for what we need at the moment—we can switch in the future when Travis is not enough.

**Comment by aperezdc** [comment]
Merged #119 instead of this Anyway thanks a lot to @psaavedra for making the SDK available and making this PR, which have been very useful :muscle:

**Comment by psaavedra** [comment]
I see. Good to know it. Then, it makes much more sense to do it with Travis
CI in this case so far.

BTW, thanks for translate the Gitlab CI file to Travis.

**Pull request closed without merging**